### DISCUSSION TITLE
How to fix the random seed?

### DISCUSSION THREAD

**Discussion by liuzuxin** [ymd-h/cpprb#12] [Q&A] [answered]
Hi,

Thanks for your nice repo! I am wondering if it is possible to fix the random seed when we sample from the replay buffer. I think it may help to ensure reproducibility of RL algorithms. Thanks.

Best,

**Comment by ymd-h** [comment, marked as answer]
Hi, @liuzuxin 
Thank you for your feedback.

We cannot fix the random seed in the cpprb.

In my opinion, reproducibility of RL algorithm should be discussed together with enough statistics and variance of the evaluation.
